Modules/ITS/src/ITSClusterTask.cxx

```diff
@@ -221,8 +222,9 @@ void ITSClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
         nClustersForBunchCrossing++;
 
       if (lay < NLayerIB) {
-
         mClusterOccupancyIB[lay][sta][chip]++;
+	int ilane =(int) chip/3;
+        mClusterOccupancyLaneIB[lay][sta][ilane]++;
```

Comment:
This one is not need, since you can look at `mClusterOccupancyIB` (see comment to header)